wiki/People/Users_with_unique_titles/id.md

```diff
@@ -154,28 +157,28 @@ Kontes osu!remix menghadiahkan gelar **osu!mixer** sebagai hadiah utama dari kon
 | ![][flag_RU] [tieff](https://osu.ppy.sh/users/89619) | [tieff](https://osu.ppy.sh/beatmaps/artists/34) |
 | ![][flag_CA] [VINXIS](https://osu.ppy.sh/users/4323406) | [VINXIS](https://osu.ppy.sh/beatmaps/artists/22) |
 
-### Pemenang kontes perancangan maskot
+### Pemenang kontes-kontes desain maskot
 
-osu!team mengadakan kontes seni grafis yang terbuka untuk komunitas osu! dan menawarkan juga kesempatan untuk merancang atau merancang ulang beberapa maskot mode game yang dimiliki osu!. Pemenang kontes ini, serta orang - orang yang rancangannya digunakan sebagai maskot resmi osu!, telah dianugerahi gelar pengguna **osu!artist**.
+Dari waktu ke waktu, osu! telah beberapa kali mengadakan kontes desain maskot yang terbuka bagi seluruh ilustrator yang tertarik untuk mendesain atau merombak tampilan maskot-maskot yang ada. osu! menganugerahkan gelar **osu!artist** kepada para ilustrator yang karyanya berhasil terpilih sebagai pemenang.
 
-- ![][flag_US] [Crowie](https://osu.ppy.sh/users/6894067) merancang **Mocha** dan membantu menghasilkan banyak karya seni grafis osu! lainnya.
-- ![][flag_ID] [JMC](https://osu.ppy.sh/users/774010) memenangkan [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2).
-- ![][flag_US] [Thievley](https://osu.ppy.sh/users/4717672) merancang ulang **Yuzu**.
-- ![][flag_ID] [xiemon](https://osu.ppy.sh/users/5203667) merancang **Mani and Mari**.
+- ![][flag_US] [Crowie](https://osu.ppy.sh/users/6894067) mendesain **Mocha** dan berbagai aset grafis lainnya yang ada di osu!.
+- ![][flag_ID] [JMC](https://osu.ppy.sh/users/774010) memenangkan kontes [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2).
+- ![][flag_US] [Thievley](https://osu.ppy.sh/users/4717672)  merombak ulang desain **Yuzu**.
+- ![][flag_ID] [xiemon](https://osu.ppy.sh/users/5203667) mendesain **Mani and Mari**.
 
 ### Hall of Famer
 
-Tiga pengguna ini dianugerahi gelar **osu! Hall of Famer** atas kontribusinya telah memainkan game ini di level tertinggi dan juga berkontribusi ketika komunitas osu! masih baru.
+osu! menganugerahkan gelar **osu! Hall of Famer** kepada tiga nama di bawah ini atas kontribusinya masing-masing dalam menjunjung tinggi standar permainan pada masa-masa awal berdirinya osu!.
 
 <!-- TODO: reasons -->
 
 - ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589)
 - ![][flag_PH] [James](https://osu.ppy.sh/users/5728)
 - ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781)
 
-### Pembuat Storyboard Elit
+### Elite Storyboarder
 
-Storyboard yang menunjukkan bakat disertakan juga cerita yang luar biasa dan mengesankan komunitas osu! secara luas akan diberi gelar **Elite Storyboarder**.
+osu! menganugerahkan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh khalayak luas.
```

Comment:
```suggestion
osu! memberikan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh banyak pemain.
```

Reply:
not really a fan of how `banyak` is being repeated like that in a single sentence here (`...banyak karya yang dikagumi oleh banyak...`) so i think i'll be leaving this as is :(